shared/src/packages/articles/libs/enums/article-improvement-suggestion-priority.enum.ts

```diff
@@ -0,0 +1,7 @@
+const ArticleImprovementSuggestionPriority = {
+  Low: 1,
+  Medium: 2,
+  High: 3,
```

Comment:
Same about **B23**,
And if you declare it in shared, then I am not sure you need to duplicate it on frontend, maybe just reuse this enum?
```suggestion
  Low: 1,
  Medium: 2,
  High: 3,
```

Reply:
Just forgot to delete it 😓